scripts/tcs_startup.sh

```diff
@@ -16,28 +16,25 @@
 
 KV_STORAGE="kv_storage"
 ENCLAVE_MANAGER="${TCF_HOME}/examples/enclave_manager/tcf_enclave_manager/enclave_manager.py"
-LISTENER="${TCF_HOME}/common/python/connectors/direct/tcs_listener/tcs_listener.py"
+LISTENER="avalon_listener"
 VERSION="$(cat ${TCF_HOME}/VERSION)"
 # Default values
-START_STOP_KV_STORAGE=0 # default if -s not passed
+START_STOP_AVALON_SERVICES=0 # default if -s not passed
 LMDB_URL="http://localhost:9090" # -l default
-
+LISTENER_URL="http://localhost:1947"
 # Trap handler
 trap 'stop_avalon_components' HUP INT QUIT ABRT ALRM TERM
 
 start_avalon_components()
 {
-    # Read Listener port from config file
-    LISTENER_PORT=`grep listener_port ${TCF_HOME}/config/tcs_config.toml \
-        | awk {'print $3'}`
-    if [ -z "$LISTENER_PORT" ]; then
-        LISTENER_PORT=1947
-    fi
-
-    if [ $START_STOP_KV_STORAGE = 1 ] ; then
+    if [ $START_STOP_AVALON_SERVICES = 1 ] ; then
         echo "Starting Avalon KV Storage $VERSION ..."
         python3 $KV_STORAGE --bind $LMDB_URL & 
         echo "Avalon KV Storage started"
+
+	echo "Starting Avalon Listener $VERSION ..."
+	$LISTENER --bind $LISTENER_URL --lmdb_url $LMDB_URL &
+	echo "Avalon Listener started"
```

Comment:
since we use a generic name START_STOP_AVALON_SERVICES , I would assume we also need to start enclave manager based on this condition. I guess that will be done we add support to start enclave manger as a separate container. Please add a comment why enclave manager should be started independent of this variable.

Reply:
ok